src/tests/interpolation/test_Quad2D.cc

```diff
@@ -0,0 +1,208 @@
+/*
+ * (C) Crown Copyright 2021 Met Office
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include "eckit/types/FloatCompare.h"
+
+#include "atlas/interpolation/element/Quad2D.h"
+#include "atlas/util/Point.h"
+
+#include "tests/AtlasTestEnvironment.h"
+
+using atlas::PointXY;
+using atlas::interpolation::element::Quad2D;
+using atlas::interpolation::method::Intersect;
+
+namespace atlas {
+namespace test {
+
+//----------------------------------------------------------------------------------------------------------------------
+
+const double relative_error = 0.0001;
+
+CASE("test_quad_area") {
+    PointXY v0(0., 0.);
+    PointXY v1(1., 0.);
+    PointXY v2(1., 1.);
+    PointXY v3(0., 1.);
+
+    Quad2D quad1(v0.data(), v1.data(), v2.data(), v3.data());
+
+    EXPECT(quad1.validate());
+
+    double area = quad1.area();
+
+    std::cout << "area " << area << std::endl;
+    EXPECT(eckit::types::is_approximately_equal(area, 1.0, relative_error));
```

Comment:
This place and all following, also in different test files:
```suggestion
    EXPECT_APPROX_EQ(area, 1.0, relative_error);
```
This will print the LHS and RHS when the EXPECT fails.
It could be that you copied something like this from an existing test, but I'd like new tests to be better. And this way you learned something new as well.